jenkins/scripts/bare_metal_lab/README.md

```diff
@@ -1,17 +1,17 @@
 # Baremetal Lab Setup
 
-This repo is created to setup the baremetal lab for metal3.
+The Bare Metal Lab needs some special treatment compared to other pipelines since it does not use VMs for the target cluster.
+This is taken care of in the `deploy-lab.yaml` playbook.
 
 ## Ansible installation
 
 `sudo pip3 install ansible`
 
 ## Running the playbook
 
-* Set 'bm-m3-lab' to the IP address of the lab in /etc/hosts.
```

Comment:
why is this removed?

Reply:
It does not seem it is used/needed. I'm guessing there used to be a `hosts` file that had `bm-m3-lab` in it but this is no longer the case. The playbook is hard coded to run on `localhost` and the pipeline is not using any inventory, just `ansible-playbook <path-to-playbook>`.